cmd/event/scrub/scrub.go

```diff
@@ -12,6 +12,9 @@ import (
 
 var all bool
 
+const scrubPushedConfirmMsg = "You are trying to remove all pushed events and their associated. This cannot be undone. Are you sure you want to proceed? [y/n]"
```

Comment:
Did you want to use the word "pushed"? Perhaps drop the word or use "registered" instead.
"associated" what? sentence is dangling.
s/associated/associated readings.

Reply:
We should keep "pushed"  ..